modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -26,11 +30,54 @@ class Configuration(object):
     :param api_key_prefix: Dict to store API prefix (e.g. Bearer)
     :param username: Username for HTTP basic authentication
     :param password: Password for HTTP basic authentication
+    :param key_id: The identifier of the cryptographic key, when signing HTTP requests.
+        An 'Authorization' header is calculated by creating a hash of select headers,
+        and optionally the body of the HTTP request, then signing the hash value using
+        a private key which is available to the client.
+    :param private_key_path: The path of the file containing a private key,
+        when signing HTTP requests.
+    :param signing_scheme: The signature scheme, when signing HTTP requests.
+        Supported value is hs2019.
+    :param signing_algorithm: The signature algorithm, when signing HTTP requests.
+        Supported values are PKCS1v15, PSS; fips-186-3, deterministic-rfc6979.
```

Comment:
Do these values already exist as constants in your imported crypto libraries?
Why not use those constants rather than strings?

Reply:
I am checking if they exist.

Reply:
I searched in the source code, it appears these strings are hard-coded. There is no constant. Marking as resolved for now.